tests/core/opcodes/test_opcodes.py

```diff
@@ -160,6 +160,15 @@ def test_add(vm_class, val1, val2, expected):
     assert result == expected
 
 
+def test_base_fee():
+    computation = run_general_computation(LondonVM)
+    computation.opcodes[opcode_values.BASEFEE](computation)
+
+    result = computation.stack_pop1_int()
```

Comment:
```suggestion
    result = computation.stack_pop1_any()
```

^ is a way to make sure that an int was pushed to the stack natively. (because it won't try to coerce the type on the way out)